pkg/compute/compute_deploy_test.go

```diff
@@ -357,6 +357,27 @@ func TestDeploy(t *testing.T) {
 				"Deployed package (service 123, version 4)",
 			},
 		},
+		{
+			name: "success with comment",
+			args: args("compute deploy --token 123 -s 123 -p pkg/package.tar.gz --version 2 --comment foo"),
+			api: mock.API{
+				GetServiceFn:      getServiceOK,
+				ListVersionsFn:    testutil.ListVersions,
+				CloneVersionFn:    testutil.CloneVersionResult(3),
```

Comment:
```suggestion
				CloneVersionFn:    testutil.CloneVersionResult(4),
```

Although, as far as mocking is concerned, it doesn't really matter what we return here, I would prefer the code to assert expectations that resemble the real life outcome. So, as `testutil.ListVersions` will return a list of three versions (where version 2 is 'locked') the real life outcome is that the locked version would be cloned and result in the output being: 

```
"Deployed package (service 123, version 4)"
```

Reply:
I agree, this was an oversight on my end.